src/components/DefinitionList/utils.ts

```diff
@@ -1,7 +1,46 @@
+import React from 'react';
+
+import {block} from '../utils/cn';
+
+import type {DefinitionListGroup, DefinitionListItem, DefinitionListItemGrouped} from './types';
+
+export const b = block('definition-list');
+
 export function isUnbreakableOver(limit: number) {
     return function (value: string): boolean {
         const posibleLines = value.split(/\s+/);
 
         return posibleLines.some((line) => line.length > limit);
     };
 }
+
+export const isGroup = (
+    item: DefinitionListItem | DefinitionListGroup,
+): item is DefinitionListGroup => 'label' in item;
```

Comment:
Checking `items` prop is more accurate, maybe we'll have the `label` prop on normal item too